hw/dv/verilator/memutil_dpi.core

```diff
@@ -0,0 +1,20 @@
+CAPI=2:
```

Comment:
If we're being picky, these files should probably move out of the hw/dv/verilator tree and into (maybe) the hw/dv/dpi tree. I'm not that bothered about that for now though!

Reply:
Yeah, I thought I'd try to get everything working in a useful example before a big file rename. But it's definitely on the list!